src/sql-parser/src/parser.rs

```diff
@@ -2042,6 +2056,45 @@ impl<'a> Parser<'a> {
         })
     }
 
+    fn parse_kafka_broker(&mut self) -> Result<WithOptionValue<Raw>, ParserError> {
+        let _ = self.consume_token(&Token::Eq);
+        let address = self.parse_literal_string()?;
+        let aws_privatelink = if self.parse_keyword(USING) {
+            self.expect_keywords(&[AWS, PRIVATELINK])?;
+            let connection = self.parse_raw_name()?;
+            let options = if self.consume_token(&Token::LParen) {
```

Comment:
This seems to have broken some tests in test/testdrive/connection-create-drop.td which expect a `[` instead of a `(`. Is this a user-facing change to no longer support square brackets?

Reply:
Probably, if we want to keep this then we'll likely have to write a migration since the Create SQL statement is saved in the catalog.

Reply:
Yeah let's definitely restore the old behavior of allowing either `[` and `(`, at least for now.

Reply:
Done

Reply:
This was not the right place actually. I left this only allowing parenthesis and adding brackets back to kafka brokers.